oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticCustomAnalyzer.java

```diff
@@ -189,6 +189,10 @@ private static List<String> loadContent(NodeState file, String name) throws IOEx
             try (BufferedReader br = new BufferedReader(content)) {
                 return br.lines()
                         .map(String::trim)
+                        // remove empty lines and comments
+                        .filter(l -> l.length() > 0 && !l.startsWith("#"))
```

Comment:
I saw a different index definition with comments that start with `|`.
E.g [stopwords.txt](https://github.com/apache/jackrabbit-oak/files/11383877/stopwords.txt)
I don't think this will protect against that case.

Reply:
I have never seen this format for both stop words and comments. I don't even understand what the  `|` means in the configuration (example: `de             |  from, of`). It might be that lucene/elastic don't complain with a configuration like this but I am not sure about the results. I can create a test with this format but I am not sure about a query that proves this configuration.

Reply:
I wonder, is this code used for stopwords as well? I thought it's just mappings... If that's the case, I think it's better to rename the method from loadContent to loadMappingFileContent or so...